datajunction-server/datajunction_server/database/partition.py

```diff
@@ -29,6 +29,9 @@ class Partition(Base):  # type: ignore
     """
 
     __tablename__ = "partition"
+    __table_args__ = (
+        Index("idx_partition_column_id", "column_id", postgresql_using="btree"),
```

Comment:
I think btree is the default, so probably no need to specify it everywhere.

Reply:
Agreed, also removed these indexes because I don't think they make a difference for the specific slow endpoints I'm looking at